### PR TITLE
Update features.md

### PR DESCRIPTION
**Pull request by HerHde**

### CODE REVIEW

File: public/docs/features.md [hunk 1]
```diff
@@ -11,121 +10,128 @@ If you experience any _issues_, feel free to report it on [**GitHub**](https://g
...
-For vim:
-> For more information, see [here](https://codemirror.net/demo/vim.html).
+### Auto-Complete
 
-## Auto-Complete:
 This editor provides full auto-complete hints in markdown.
+
 - Emojis: type `:` to show hints.
-- Code blocks: type ` ``` ` and plus a character to show hint. <i hidden>```</i>
+- Code blocks: type ` ``` ` and plus a character to show hint.
```
Comment (ccoenen) [inline public/docs/features.md]:
```diff
-and plus a character
+and another character
```

File: public/docs/features.md [hunk 2]
```diff
@@ -11,121 +10,128 @@ If you experience any _issues_, feel free to report it on [**GitHub**](https://g
...
-**Desktop & Tablet**
+## Workspace
+
+### Modes
+
+#### Desktop & Tablet
 
 <i class="fa fa-edit fa-fw"></i> Edit: See only the editor.
 <i class="fa fa-eye fa-fw"></i> View: See only the result.
 <i class="fa fa-columns fa-fw"></i> Both: See both in split view.
```
Comment (ccoenen) [inline public/docs/features.md]: Does anyone else wish they were in the order of the toolbar? I know that it's somewhat confusing to talk about "both" when you haven't introduced both, but somehow this order bugs me.

Suggestion:

```diff
-See both in split view.
+See editor and result at the same time.
```
(and then put it as the middle bullet point?)

File: public/docs/features.md [hunk 3]
```diff
@@ -11,121 +10,128 @@ If you experience any _issues_, feel free to report it on [**GitHub**](https://g
...
+### Night Mode
+
 When you are tired of a white screen and like a night mode, click on the little moon <i class="fa fa-moon-o"></i> and turn on the night view of CodiMD.
 
 The editor view, which is in night mode by default, can also be toggled between night and day view using the the little sun<i class="fa fa-sun-o fa-fw"></i>.
 
-## Image Upload:
+### Image Upload
+
 You can upload an image simply by clicking on the camera button <i class="fa fa-camera"></i>.
```
Comment (ccoenen) [inline public/docs/features.md]: that camera button no longer exists, It is now 
upload button editor

I'm not sure if this is included in fontawesome, but @SISheogorath may know?

File: public/docs/features.md [hunk 4]
```diff
@@ -11,121 +10,128 @@ If you experience any _issues_, feel free to report it on [**GitHub**](https://g
...
 Alternatively, you can **drag-n-drop** an image into the editor. Even **pasting** images is possible!
-This will automatically upload the image to **[imgur](http://imgur.com)**, **[Amazon S3](https://aws.amazon.com/s3/)**, **[Minio](https://minio.io)** or **local filesystem**, nothing to worry about. :tada:
+This will automatically upload the image to **[imgur](http://imgur.com)**, **[Amazon S3](https://aws.amazon.com/s3/)**, **[Minio](https://minio.io)** or the **local filesystem** (depending on the instance's configuration), nothing to worry about. :tada:
 ![imgur](https://i.imgur.com/9cgQVqD.png)
```
Comment (ccoenen) [inline public/docs/features.md]: please add a blank line before this one, the image currently sticks directly to the paragraph (an error that was in the earlier version too, not your fault!)

File: public/docs/features.md (hunk 3 above)
Comment (SISheogorath) [reply public/docs/features.md]: should be `fa-upload` instead of `fa-camera`

Comment (ccoenen) [reply public/docs/features.md]: whoops, I always mix those up.

File: public/docs/features.md (hunk 1 above)
Comment (HerHde) [reply public/docs/features.md]:
```suggestion
- Code blocks: type ` ``` `, followed by another character to show syntax highlighting suggestions.
```

File: public/docs/features.md (hunk 4 above)
Comment (HerHde) [reply public/docs/features.md]:
```suggestion

![imgur](https://i.imgur.com/9cgQVqD.png)
```

File: public/docs/features.md (hunk 1 above)
Comment (HerHde) [reply public/docs/features.md]: What do you think?

Comment (ccoenen) [reply public/docs/features.md]: I like it!

**Pull request merged**